packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -476,6 +476,8 @@ export default function DimensionsPanel( {
 				>
 					<HStack alignment="flex-end" justify="flex-start">
 						<UnitControl
+							// TODO: Switch to `true` (40px size) if possible
+							__next40pxDefaultSize={ false }
```

Comment:
@WordPress/gutenberg-design Do you have a preference on how to upsize the "Content" and "Wide" controls? We can for example break them up into two rows, or remove the icon. There is hint text below it that applies to both controls, which may be harder to visually grasp when broken up into two rows.

<img src="https://github.com/user-attachments/assets/07d80b80-c336-463a-a1d9-8c295caa4263" alt="Layout controls" width="273">

Reply:
I recently tinkered with this mockup:

<img width="484" alt="dimensions" src="https://github.com/user-attachments/assets/c5584f0d-4271-473a-9d68-90be708eb736">

What do you think?

Reply:
One drawback with this approach is that we now have decorative icons situated in a location where you often find interactive icon buttons. In other words those icons look clickable. 

Could they appear inside the field instead, as is standard practise with similar fields?

<img width="558" alt="Screenshot 2024-08-15 at 15 15 51" src="https://github.com/user-attachments/assets/0ba0f383-f8f9-43dc-a5ef-77ad7bd86e7a">

Reply:
Maybe, though to me they also look clickable there :). They also wouldn't line up with other icons in the panel, though that's secondary since it's alread an issue.

Reply:
I find the InputControl prefix slot is usually used for decorative items, whereas the label prefix is usually for interactive items, yes. Ideally I'd like to put it in the prefix slot, but not sure if it'll fit in a two-column layout. I can try.

Reply:
It was pretty tight but not impossible 🙁 Assuming this input will take four digits maximum with the first digit being a "1". May clip a little depending on the font. Also the left padding on the icon needed to be an off-grid value (6px).

<img src="https://github.com/user-attachments/assets/8c31dcec-d09e-4163-947d-403aad949d41" alt="Upsized Dimensions controls" width="272">

Should we go with this then?

Reply:
It feels like the more re-usable pattern to me.

I appreciate it'd be separate, but some more space could be saved by removing the padding unit select.

Reply:
I think this is the tightest we can go in the general case because that select supports `rem` units. But it can hug better in certain cases using the `field-sizing` property, similar to #63265.

Reply:
In Chrome on Windows OS, the default width of the scrollbar is 17px, and it has a physical size. The editor sidebar is 280px wide, so when the vertical scrollbar is present, the panel is 263px wide.

At this width, some of the numbers get cut off, so it would be nice to remove the icon or adjust the padding:

![image](https://github.com/user-attachments/assets/4e7b18bc-1363-42fd-b6f2-e1f2924c6584)

Reply:
Separately, should we use this icon for Content instead?

<img width="36" alt="image" src="https://github.com/user-attachments/assets/e1c278bd-95f0-404b-93a8-4bb3cfa258ba">

Reply:
> I think this is the tightest we can go in the general

Can't we remove the padding? It only seems to be there to accommodate the outline on hover. That seems to be a custom treatment, so could easily be replaced with something else like underlining.

The gap between the value and the select could be reduced to `4px` to save some more space. 

Finally (again acknowledging it'd be separate), I've been thinking for a little while that we should reduce the padding on the inputs. `16px` feels a bit large to me...

| 16px | 12px |
| --- | --- |
| <img width="341" alt="Screenshot 2024-08-16 at 14 00 23" src="https://github.com/user-attachments/assets/a1ee7a7b-d035-4127-bdfa-2aac44a2622d"> |  <img width="355" alt="Screenshot 2024-08-16 at 14 01 50" src="https://github.com/user-attachments/assets/1152d309-799c-417a-a2d2-07968be34d97"> |

Reply:
Judging from @t-hamano's screenshot, we need to make way for at least 12 more pixels, which is going to require changes at the design system level if we want it to work. So for this PR I can either abandon upsizing these particular instances, or break it into two rows until we figure it out. If nobody has a preference, I'll be doing the latter.

Reply:
What do you think of either of these approaches?

Remove the prefix icon. The screenshot below shows how the UI looks when scrollbars are visible on Windows OS:

![image](https://github.com/user-attachments/assets/e21ed7e9-dfb8-4aef-a96b-23b451bf6faa)

Show an icon to the right of the label, as suggested in [this comment](https://github.com/WordPress/gutenberg/pull/64520#discussion_r1718015919). However, this mockup might be a bit too much of a code change to implement in this PR.

Of course, breaking it into two rows would also be fine for me.

Reply:
I personally find these icons really helpful in understanding what "content" and "wide" are referring to, so I'm hesitant to remove them. And I'm somewhat strongly opposed to mixing interactive and non-interactive items in the label suffix, unless we style them differently. In any case I think this kind of thing should be considered and clearly defined in our design system.

So I think breaking it in two rows would be the least disruptive for now.

Reply:
I understand, I think two rows would be fine 👍

Reply:
I'm not sure about the two rows, being on one row implies these are related, which they are. I think this is worth solving in design before we move that much farther.

Did you consider the 12px padding left/right with a smaller unit selector? The gaps in https://github.com/WordPress/gutenberg/pull/64520#discussion_r1719172857 should be possible to reduce quite a bit.

Reply:
Ok, if two rows are also not great, I'll leave these controls as is for this PR. The required changes are out of scope.

I'm on board with reducing the input paddings to 12px though, as long as we do it separately and the Figma files are updated 👍 As for the unit selector, I'm not sure we have much to reduce unless we do something a bit more drastic, since this is what we're dealing with:

<img width="86" alt="Unit selector for rem units" src="https://github.com/user-attachments/assets/250a8787-854c-404e-810a-48da7e7b9361">

Reply:
Ooh, the [new Figma file](https://www.figma.com/design/804HN2REV2iap2ytjRQ055/Core-System-Library?node-id=991-34623) already does have tighter margins, perfect! Will reflect in code then 👍

Reply:
Input padding decrease proposed in #64708.

Reply:
Unit select dropdown changes proposed in #64712.